website/docs/r/packet_capture.html.markdown

```diff
@@ -0,0 +1,188 @@
+    name = "osdisk"
+    caching = "ReadWrite"
+    create_option = "FromImage"
+    managed_disk_type = "Standard_LRS"
+  }
+
+  os_profile {
+    computer_name = "pctest-vm"
+    admin_username = "testadmin"
+    admin_password = "Password1234!"
+  }
+
+  os_profile_linux_config {
+    disable_password_authentication = false
+  }
+}
+
+resource "azurerm_virtual_machine_extension" "test" {
+  name                       = "network-watcher"
+  location                   = "${azurerm_resource_group.test.location}"
+  resource_group_name        = "${azurerm_resource_group.test.name}"
+  virtual_machine_name       = "${azurerm_virtual_machine.test.name}"
+  publisher                  = "Microsoft.Azure.NetworkWatcher"
+  type                       = "NetworkWatcherAgentLinux"
+  type_handler_version       = "1.4"
+  auto_upgrade_minor_version = true
+}
+
+
+resource "azurerm_storage_account" "test" {
+  name = "pctestsa"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  location = "${azurerm_resource_group.test.location}"
+  account_tier = "Standard"
+  account_replication_type = "LRS"
+}
+
+resource "azurerm_packet_capture" "test" {
+  name                 = "pctestcapture"
+  network_watcher_name = "${azurerm_network_watcher.test.name}"
+  resource_group_name  = "${azurerm_resource_group.test.name}"
+  target_resource_id   = "${azurerm_virtual_machine.test.id}"
+
+  storage_location {
+    storage_account_id = "${azurerm_storage_account.test.id}"
+  }
+
+  depends_on = ["azurerm_virtual_machine_extension.test"]
+}
+```
+
+~> **NOTE:** This Resource requires that [the Network Watcher Virtual Machine Extension](https://docs.microsoft.com/azure/network-watcher/network-watcher-packet-capture-manage-portal#before-you-begin) is installed on the Virtual Machine before capturing can be enabled which can be installed via [the `azurerm_virtual_machine_extension` resource](virtual_machine_extension.html).
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name to use for this Packet Capture. Changing this forces a new resource to be created.
+
+* `network_watcher_name` - (Required) The name of the Network Watcher. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group in which the Network Watcher exists. Changing this forces a new resource to be created.
+
+* `target_resource_id` - (Required) The ID of the Resource to capture packets from. Changing this forces a new resource to be created.
+
+~> **NOTE:** Currently only Virtual Machines ID's are supported.
+
+* `maximum_bytes_per_packet` - (Optional) The number of bytes captured per packet. The remaining bytes are truncated. Defaults to `0` (Entire Packet Captured). Changing this forces a new resource to be created.
+
+* `maximum_bytes_per_session` - (Optional) Maximum size of the capture in Bytes. Defaults to `10737441824` (1GB). Changing this forces a new resource to be created.
```

Comment:
Typo: default does not match what is defined in the schema: `1073741824`

Reply:
confirmed in the portal this should be `1073741824` - will fix

Reply:
fixed